scripts/update_gtfs.sh

```diff
@@ -2,4 +2,3 @@
 set -e
 
 curl -o var/mbta-ma-us.gtfs.zip "${MBTA_GTFS_URL:-"https://mbta-gtfs-s3.s3.amazonaws.com/google_transit.zip"}"
-curl -o var/massport-ma-us.gtfs.zip https://data.trilliumtransit.com/gtfs/massport-ma-us/massport-ma-us.zip
```

Comment:
Yeah, I thought referencing environment variables wouldn't work here but it turns out it works! I've since created consistency by defining all the external resources in this config! ✨